.github/workflows/main.yaml

```diff
@@ -34,95 +34,25 @@ jobs:
       run: poetry install
 
     - name: Run tests
+      run: make test
+
+    - name: Run Smoke Test (mkdocs)
       run: |
-        # test is not working
-        # make test
-        cookiecutter --output-dir /tmp/ --no-input .
-        cd /tmp/osl-python-package
-        mamba env create --file conda/dev.yaml
-        conda activate osl-python-package
-        poetry install
-        pre-commit install
-        make docs
-        make build
+        chmod +x tests/smoke/mkdocs.sh
+        ./tests/smoke/mkdocs.sh
+
+    - name: Run Smoke Test (sphinx)
+      run: |
+        chmod +x tests/smoke/sphinx.sh
```

Comment:
you don't need to use `chmod` here. 
try to run it locally.

if it doesn't work, np and call the script with bash: `bash ./tests/smoke/sphinx.sh`